Pages/Crime.cshtml

```diff
@@ -53,7 +53,7 @@
 foreach (QuickTypeCrimes.Crime crime in crimes)
             {
         <tr>
-            <td>@crime.DateReported</td>
+            <td>@crime.DateReported.ToString("d")</td>
```

Comment:
Changed the field type to incorporate Date values instead of the Date Time values for the tabular data. Reduces the technical debt by improving the User Experience.
Source
https://www.w3.org/TR/tabular-data-model/
http://www.cheat-sheets.org/saved-copy/msnet-formatting-strings.pdf

Reply:
Huh, neat.